challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map { |word| word.capitalize }
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select { |word| word == "dog" }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.find_all { |word| word != "dog" }
```

Comment:
[Enumerable#reject](http://ruby-doc.org/core-2.0.0/Enumerable.html#method-i-reject) is also available.